alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunnerService.kt

```diff
@@ -138,6 +142,21 @@ object MonitorRunnerService : JobRunner, CoroutineScope, AbstractLifecycleCompon
             monitorCtx.maxActionableAlertCount = it
         }
 
+        monitorCtx.triggerMaxActions = ALERTING_TRIGGER_MAX_ACTIONS.get(monitorCtx.settings)
+        monitorCtx.triggerTotalMaxActions = ALERTING_TRIGGER_TOTAL_MAX_ACTIONS.get(monitorCtx.settings)
+        monitorCtx.clusterService!!.clusterSettings.addSettingsUpdateConsumer(
+            ALERTING_TRIGGER_MAX_ACTIONS,
+            ALERTING_TRIGGER_TOTAL_MAX_ACTIONS
+        ) { maxActions: Int, totalMaxActions: Int ->
+            if (maxActions > totalMaxActions) {
+                throw IllegalArgumentException(
+                    "The limit number of a single trigger should not be greater " +
+                        "than that of the overall trigger $maxActions $totalMaxActions"
+                )
+            }
+            monitorCtx.triggerMaxActions = maxActions
+            monitorCtx.triggerTotalMaxActions = totalMaxActions
```

Comment:
Since both the settings are Dynamic, what happens if the new max action settings is below the existing monitor triggers, with a higher value? 

Can we rather enforce a validation check and fail update with the right message instead, to enforce user to reduce the triggers first?